### ISSUE TITLE
No pleasant use of the handles

### ISSUE THREAD

**Issue by friebetill** [memspace/zefyr#187]
## Problem
If one compares the native selection on Android (e.g. with Chrome on a web page) with that of Zefyr, one notices how difficult it is to select the handles of the text selection in Zefyr.

The problem is even worse if you add another gesture, e.g. a `TabController`.

## Steps to Reproduce
I've extended the example in [this forked repository](https://github.com/friebetill/zefyr/tree/master) to illustrate the problem. 

## Logs
```
[✓] Flutter (Channel unknown, v1.9.1+hotfix.5, on Mac OS X 10.14.6 18G95, locale en-DE)
    • Flutter version 1.9.1+hotfix.5 at /Users/maxwell/Software/flutter
    • Framework revision 1aedbb1835 (5 days ago), 2019-10-17 08:37:27 -0700
    • Engine revision b863200c37
    • Dart version 2.5.0


[✓] Android toolchain - develop for Android devices (Android SDK version 28.0.3)
    • Android SDK at /Users/maxwell/Library/Android/sdk
    • Android NDK location not configured (optional; useful for native profiling support)
    • Platform android-29, build-tools 28.0.3
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_202-release-1483-b49-5587405)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 10.2)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 10.2, Build version 10E125
    • CocoaPods version 1.7.5

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 3.5)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 40.0.2
    • Dart plugin version 191.8423
    • Java version OpenJDK Runtime Environment (build 1.8.0_202-release-1483-b49-5587405)

[✓] VS Code (version 1.39.1)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.5.1

[✓] Connected device (2 available)
    • Nexus 5X • 0112199c644ec244 • android-arm64  • Android 8.1.0 (API 27)
    • Chrome   • chrome           • web-javascript • Google Chrome 77.0.3865.120
```

## Help
If you like, I can take a close look at the problem and try to find a solution. In [this post](https://medium.com/flutter-community/combining-multiple-gesturedetectors-in-flutter-26d899d008b2), the developer presents a solution that might fix the problem.

**Comment by TutOsirisOm** [comment]
I've been looking into this for the last day or so. I haven't found anything else on it till you posted this. I'll update if I come up with a solution.

**Comment by TutOsirisOm** [comment]
@friebetill Thanks! I implemented that solution just now in the current version. It seems to work better, but definitely not the end result I'm going for yet. I'm using the ZefyrField in a TabView and most of the time the selection handles don't move when just trying to select a word or two out of a single line sentence, before and after applying this fix.

**Comment by a9udn9u** [comment]
Having the same issue, it happens even without other widgets competing gesture handlings. I tested in a basic app with nothing but a Zefyr editor.

![screenrecord](https://i.imgur.com/s0MH3sL.gif)

This was on my phone, it happens in the iOS simulator and the Android emulator as well.

**Comment by pulyaevskiy** [comment]
I believe selection handles UX has been improved in `1.0.0-dev` branch and is now released in `1.0.0-rc.1`. Feel free to reopen and submit a new issue if there are still problems with selection handles.
